src/main/java/com/google/growpod/data/ContainsPlant.java

```diff
@@ -0,0 +1,67 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.growpod.data;
+
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.Entity.Builder;
+import com.google.cloud.datastore.Key;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+/** Relates a garden and a plant through a contains relation. */
+@Data
+@AllArgsConstructor(access = AccessLevel.PUBLIC)
+public class ContainsPlant {
```

Comment:
Hmm. If the only point of this entity type is to map Garden/Plant, what would you think about giving it a name more like: `GardenPlantRelation`

Reply:
My thought pattern behind naming the relationship classes was as follows
```
"A garden" ContainsPlant "plant"
"A garden" HasMember "user"
```

The exact name you suggest, `GardenPlantRelation`, does not describe the nature of the relationship between `Garden`s and `Plant`s. Nevertheless, I will consider renaming these relationship classes.